server/src/main/java/org/opensearch/threadpool/ThreadPool.java

```diff
@@ -222,10 +228,99 @@ public Collection<ExecutorBuilder> builders() {
         Setting.Property.NodeScope
     );
 
+    public static final Setting<Settings> CLUSTER_THREAD_POOL_SIZE_SETTING = Setting.groupSetting(
+        "cluster.thread_pool.",
```

Comment:
Why does this look incomplete?

Reply:
That is  how group settings are defined .